coriolis/conductor/rpc/client.py

```diff
@@ -277,11 +277,18 @@ def task_event(self, ctxt, task_id, level, message):
         self._client.cast(ctxt, 'task_event', task_id=task_id, level=level,
                           message=message)
 
-    def task_progress_update(self, ctxt, task_id, current_step, total_steps,
-                             message):
-        self._client.cast(ctxt, 'task_progress_update', task_id=task_id,
-                          current_step=current_step, total_steps=total_steps,
-                          message=message)
+    def add_task_progress_update(self, ctxt, task_id, total_steps, message):
+        self._client.cast(ctxt, 'add_task_progress_update', task_id=task_id,
```

Comment:
What is the difference between the `cast` and `call` methods of the RPC client libs?